examples/logging-prometheus/README.md

```diff
@@ -0,0 +1,219 @@
+# DeepSparse + Prometheus/Grafana
+
+This is a simple example that shows you how to connect DeepSparse Logging to the Prometheus/Grafana stack.
+
+#### There are four steps:
+- Configure DeepSparse Logging to log metrics in Prometheus format to a REST endpoint
```

Comment:
Not very well versed here, but are we for sure using REST? The fact that we are using HTTP API does not automatically imply REST. Not sure.